defaults/test_default.yaml

```diff
@@ -206,7 +206,7 @@ stress_image:
   ndbench: 'scylladb/hydra-loaders:ndbench-jdk8-20210720'
   ycsb: 'scylladb/hydra-loaders:ycsb-jdk8-20220918'
   nosqlbench: 'scylladb/hydra-loaders:nosqlbench-4.15.49'
-  cassandra-stress: '' # default would be same version as scylla under test
+  cassandra-stress: 'scylladb/scylladb-releng:5.4.0-dev-0.20230711.e1ef3b2ced5d-x86_64' # default would be same version as scylla under test
```

Comment:
temporary or we'll live with that until i'ts on scylla master?

Reply:
temporary for testing, I'll take it out before merge

Reply:
removed